pkg/server/BUILD.bazel

```diff
@@ -171,6 +171,8 @@ go_library(
         "//pkg/util/timeutil",
         "//pkg/util/tracing",
         "//pkg/util/tracing/tracingpb",
+        "//pkg/util/tracingservice",
```

Comment:
I think just a slightly better package structure could be defining this entire package under pkg/util/tracing/service -- it's find for a nested package to depend on a top-level directory, we use this pattern in other places. Importing packages could import it as tracingservice or whatever. I just want to try and avoid two top-level tracing packages, when they could be just the one with multiple sub-packages.

Reply:
Also see comment elsewhere about the `tests` subpackage, I don't think we need it.

Reply:
done, thanks.